### PR TITLE
Removed GA3 sunset note and added no hits note for Cloud, #PG-3912

### PR DESCRIPTION
**Pull request by AltamashShaikh**
### Description:

Removed GA3 sunset note and added no hits note for Cloud
Fixes: #PG-3912

### Review

* [ ] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [ ] [Potential edge cases thought about](https://developer.matomo.org/guides/pull-request-reviews#potential-edge-cases-thought-about) (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] [Usability review done](https://developer.matomo.org/guides/pull-request-reviews#usability-review-done) (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] [Security review done](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] [Wording review done](https://developer.matomo.org/guides/pull-request-reviews#translations-wording-review-done)
* [ ] [Code review done](https://developer.matomo.org/guides/pull-request-reviews#code-review-done)
* [ ] [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [ ] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)
* [ ] [Developer changelog updated if needed](https://developer.matomo.org/guides/pull-request-reviews#developer-changelog-updated-if-needed)
* [ ] [Documentation added if needed](https://developer.matomo.org/guides/pull-request-reviews#documentation-added-if-needed)
* [ ] Existing documentation updated if needed

### CODE REVIEW

File: lang/en.json [hunk 1]
```diff
@@ -158,6 +156,7 @@
         "StreamIdFilterHelpText": "The Stream ID for which you want to import data. %1$sLearn more.%2$s %3$sNote:%4$s Leave this field blank to import data from all streams.",
         "RecommendationTitle": "Import Google Analytics data to Matomo (recommended for you)",
         "RecommendationText": "We have detected Google Analytics on your site, so you can set up Matomo by importing your Google Analytics data.",
-        "RecommendationButton": "Import data"
+        "RecommendationButton": "Import data",
+        "NoteImportNotCountedAsHit": "%1$sNote:%2$s Data imported through the Google Analytics Import do not count towards your hits quota."
```
Comment (snake14) [inline lang/en.json]: @Stan-vw @gillnd Data is plural in this context and would make more sense to use `does`, correct?
```suggestion
        "NoteImportNotCountedAsHit": "%1$sNote:%2$s Data imported through the Google Analytics Import does not count towards your hits quota."
```

**Review by snake14 — COMMENTED** [review]
Looks good. I just had the one question about the wording of the new note.

I'm guessing that we'll likely have to update the ConnectAccounts UI screenshots?

File: lang/en.json (hunk 1 above)
Comment (gillnd) [reply lang/en.json]: > @Stan-vw @snake14  Data is plural in this context and would make more sense to use `does`, correct?

Actually both are technically correct 'data' plural agrees with 'do' but 'data' singular can be a collective noun and uses 'does'

Comment (gillnd) [reply lang/en.json]: @snake14 This just reads better: Data imported through the Google Analytics Import does not count towards your hits quota

Comment (snake14) [reply lang/en.json]: Thank you @gillnd My reasoning was off, but it just felt like `does` made more sense. :smile:

Comment (AltamashShaikh) [reply lang/en.json]: Added as suggested :+1:

**Comment by AltamashShaikh** [comment]
@snake14 Yes will re run the tests for ConnectAccounts after merging this

**Pull request merged**